go-controller/pkg/ovn/ovn.go

```diff
@@ -795,11 +797,11 @@ func (oc *Controller) WatchNamespaces() {
 		},
 	}, oc.syncNamespaces)
 	klog.Infof("Bootstrapping existing namespaces and cleaning stale namespaces took %v", time.Since(start))
-	// TODO(FF): make this function return failures
 	if err != nil {
 		klog.Errorf("Failed to watch namespaces err: %v", err)
+		return err
 	}
-	// return err
```

Comment:
oh looks like you removed this from the first commit

Reply:
ah, yes, I can merge the 2 commits into 1 if that helps.